### PR TITLE
Mako: introduce 3.7 support

### PR DESCRIPTION
**Pull request by extrowerk**
My interpretation, any constructive idea?

### CODE REVIEW

**Comment by Begasus** [comment]
Maybe bump to 1.1.0? (not tested here)

**Comment by waddlesplash** [comment]
Looks OK?

File: dev-python/mako/mako-1.0.3.recipe [hunk 1]
```diff
@@ -53,6 +53,12 @@ PROVIDES_python="$PROVIDES_python
 REPLACES_python="
 	python_mako
 	"
+PROVIDES_python36="$PROVIDES_python3
```
Comment (korli) [inline dev-python/mako/mako-1.0.3.recipe]: Shouldn't it be $PROVIDES_python36 ?

**Comment by korli** [comment]
bumped to 1.1.0.

**Pull request closed without merging**